### ISSUE TITLE
policies and guidance

### ISSUE THREAD

**Issue by victoriafraser** [kentcc/kelsi#10]
Am trying to link to and preview the EHC Plans page and being told that 

Page not found - 404

http://beta.kelsi.org.uk/policies-and-guidance/educational-health-and-social-care-plans-guidance
http://www.kent.gov.uk/_kelsi-beta/policies-and-guidance/educational-health-and-social-care-plans-guidance

**Comment by CharlesCarnell** [comment]
I've investigated this issue with Victoria, and have come to the working conclusion that the issue is due to what was thought to be an unrelated DNS resolution error.

I'll further investigate once the DNS records have been updated to confirm if related. When testing, the page once again begins to work once signed out of Squiz. When signed into Squiz, the preview function shows you a page in a rolled back state, with the preview page inside Squiz referencing being in a currently active scrollback state.

I however could not see an option to revert the rollback state in the details page, where this functionality is typically located. I've raised a support ticket with Squiz, Squiz have raised an internal ticket in their bug tracker to further investigate the issue. I'll re-address this when Squiz return with more information.
